skbio/io/format/genbank.py

```diff
@@ -788,63 +789,6 @@ def _parse_interval(loc_str, length):
 
```

Comment:
In retrospect, the my choices of naming in line 755 seems not great. If I do it now, I would choose `strand`, `partial`. `strand` is `-`, `+`; `partial` is `00`, `01`, `11`, or `10` (with left digit indicates left partial, right indicates right partial). So it more intuitive and consistent with prodigal's output. what do you think? we can leave it for the next PR as well.